contracts/strategies/matic/multi/StrategyMaiBal.sol

```diff
@@ -17,7 +17,7 @@ import "../../../base/SlotsLib.sol";
 /// @title MAI->BAL Multi Strategy Polygon Implementation
 /// @author belbix, bogdoslav
 contract StrategyMaiBal is MaiBalStrategyBase {
-  using SlotsLib for bytes32;
+  using SlotsLib for uint;
```

Comment:
remove

Reply:
It needed for _MAI_STABLECOIN_PIPE_SLOT

Reply:
I going to remove it and replace to internal functions for do not update it when pipes are replaced